test/e2e/metering_manual_install_tests.go

```diff
@@ -3,22 +3,25 @@ package e2e
 import (
 	"context"
 	"fmt"
+	"github.com/kube-reporting/metering-operator/pkg/deploy"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/api/storage/v1beta1"
```

Comment:
Is there a reason we can't use the v1 version of the storage API and need to use the v1beta1 versioning?
```suggestion
	storagev1beta1 "k8s.io/api/storage/v1beta1"
```

Reply:
It threw an error when I tried.

Reply:
Can you expand on what error it threw? It looks like the `StorageClass` is a valid object in the v1 package: https://godoc.org/k8s.io/api/storage/v1#StorageClass